applications/RomApplication/custom_strategies/rom_builder_and_solver.h

```diff
@@ -1,765 +1,783 @@
-//    |  /           |

```

Comment:
I don't know if my GitHub got crazy but all the lines appear highlighted as changed, and I think this is not the case. Having said this, and considering how crucial this class is for the entire ROM workflow, the file must be fixed in order to only highlight the real changes.